src/core/data.jl

```diff
@@ -265,7 +482,7 @@ end
 
 ""
 function propagate_damage_status!(data::Dict{String,<:Any})
```

Comment:
Should we generalize this function to support all component types?  Namely `load`, `shunt`, `switch`, `dcline`.

Reply:
I can be generalized, but repair variables are currently not created for loads, shunts, switches, or dclines.

Reply:
Aaa, this is good for now then.